src/Domain/FaceNormal.hpp

```diff
@@ -128,5 +169,55 @@ struct InterfaceCompute<Tags::BoundaryDirectionsExterior<VolumeDim>,
   using argument_tags = tmpl::list<Tags::Interface<dirs, Mesh<VolumeDim - 1>>,
                                    Tags::ElementMap<VolumeDim, Frame>>;
 };
+
+template <size_t VolumeDim>
+struct InterfaceCompute<Tags::BoundaryDirectionsExterior<VolumeDim>,
+                        UnnormalizedFaceNormalMovingMeshCompute<VolumeDim>>
+    : db::PrefixTag,
+      db::ComputeTag,
+      Tags::Interface<
+          Tags::BoundaryDirectionsExterior<VolumeDim>,
+          Tags::UnnormalizedFaceNormal<VolumeDim, Frame::Inertial>> {
+  using dirs = BoundaryDirectionsExterior<VolumeDim>;
+
+  static std::string name() noexcept {
```

Comment:
I think you can add `using base = ...` and remove this function. Then the name is consistent with the other interface tags.

Reply:
Then I should change the existing one too, which I don't really want to do... At least this way the implementations are consistent in a file.

Reply:
@nilsleiffischer during my recent tag cleanup PRs, I intentionally skipped the Domain directory so as not to cause conflicts with the moving mesh code (something I discussed with @nilsdeppe).  I plan to do a cleanup PR for Domain tags once moving mesh is in.